scripts/us_census/pep/us_pep_sex/process.py

```diff
@@ -187,34 +202,41 @@ def _national_2000_2010(file_path: str) -> pd.DataFrame:
     Returns:
         df (pd.DataFrame) : cleaned dataframe.
     """
-    df = pd.read_csv(file_path, thousands=',', skiprows=4, header=None)
-    df.columns = [
-        'SEX', 'April2000', '2000', '2001', '2002', '2003', '2004', '2005',
-        '2006', '2007', '2008', '2009', 'April2010', '2010'
-    ]
-    df = df.query('SEX=="MALE" or SEX=="FEMALE"')
-    # dropping unwanted columns
-    df = df.drop(columns=['April2000', 'April2010', '2010'])
-    df = df.replace(
-        {'SEX': {
-            'MALE': 'Count_Person_Male',
-            'FEMALE': 'Count_Person_Female'
-        }})
-    # replacing rows with columns
-    # making the first row as column name
-    # to get all dataframe in one formate
-    df = df.transpose().reset_index()
-    df.columns = df.iloc[0]
-    df = df[1:]
-    df.insert(0, 'geo_ID', 'country/USA', True)
-    df = df.fillna(-1)
-    float_col = df.select_dtypes(include=['float64'])
-    for col in float_col.columns.values:
-        df[col] = df[col].astype('int64')
-        df[col] = df[col].astype("str").str.replace("-1", "")
-    df.rename(columns={'SEX': 'Year'}, inplace=True)
-    df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
-    return df
+    try:
+        df = pd.read_csv(file_path, thousands=',', skiprows=4, header=None)
+        df.columns = [
```

Comment:
Consider implementing a more dynamic approach to identify the required columns instead of hardcoding their order.?

Reply:
We have to assume the position of the columns [rows and cols] anyway. So it has been hard coded like other places.
Please be informed it has been handled with try and catch block anyway.

Reply:
Kuru, could you add a comment to the script explaining the reason for fixing the column order? This will help future developers understand the rationale behind the change